www-servers/automx2/automx2-2019.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_7 )
```

Comment:
python3.6 is still the profile default so you should primarily ensure support for that.

Reply:
As I wrote automx2, I can tell you without a doubt that it does not support Python 3.6. ;-)